### ISSUE TITLE
Add 'edit this page' link on codefor.de

### ISSUE THREAD

**Issue by saerdnaer** [okfde/2013.archiv.codefor.de#423]
I think it would be easier for new people to edit/create pull requests when there is an link to the coresponding file on github. For example on http://codefor.de/muenchen/ there should be a link to https://github.com/okfde/codefor.de/blob/gh-pages/muenchen/index.html

**Comment by MarHai** [comment]
Good idea but I think they need to be separated according to the page sections. So for codefor.de/muenchen it would be ...
- General/Map location: https://github.com/okfde/codefor.de/blob/gh-pages/muenchen/index.html
- Termine: https://github.com/okfde/codefor.de/blob/gh-pages/_termine/OK_Lab_Munich.md
- Projekte: https://github.com/okfde/codefor.de/tree/gh-pages/projekte (every project has an own file, ours are [mostly] named [year]-[month]-[day]-muc-[keyword])
- Mitglieder: https://github.com/okfde/codefor.de/blob/gh-pages/_labs/muenchen.yml

**Comment by gitviola** [comment]
@saerdnaer @MarHai maybe we start describing how to edit each page section in the [project Wiki](https://github.com/okfde/codefor.de/wiki) so we can link to it on every page?

Also, we could ask the users what section they want to edit and link to the right file.

**Comment by saerdnaer** [comment]
I don't think that an explanation in project wiki is essential. Once you now where to look, the yaml files are more or less self explaining. In my view, the next step is to find places for such edit links, where they don't disturb the general reader.

**Comment by saerdnaer** [comment]
@schurig Can you remove the 'waiting for response'-tag?
